SPARQLWrapper/SPARQLExceptions.py

```diff
@@ -46,23 +49,25 @@ class EndPointInternalError(SPARQLWrapperException):
     Exception type for Internal Server Error responses. Usually HTTP response status code ``500``.
     """
 
-    msg = "endpoint returned code 500 and response"
+    msg = "The endpoint returned the HTTP status code 500"
 
 
 class QueryBadFormed(SPARQLWrapperException):
     """
     Query Bad Formed exception. Usually HTTP response status code ``400``.
     """
 
-    msg = "a bad request has been sent to the endpoint, probably the sparql query is bad formed"
+    msg = "A bad request has been sent to the endpoint: it is probably the SPARQL query is badly formed"
```

Comment:
This isn't proper English, is it?

Reply:
should be: "A bad request has been sent to the endpoint: probably the SPARQL query is badly formed"